src/vcf/record.cpp

```diff
@@ -204,7 +204,9 @@ namespace ebi
                         check_field_cardinality(field.second, values, key_values["Number"], 2); // TODO Assumes ploidy=2
                         check_field_type(field.second, values, key_values["Type"]);
                     } catch (Error *ex) {
-                        throw new InfoBodyError{line, "Info " + key_values["ID"] + "=" + ex->get_raw_message()};
+                        throw new InfoBodyError{line,
+                                                "Info " + key_values["ID"] + "=" + ex->get_raw_message(),
```

Comment:
no, for a vcf with this:
```
##INFO=<ID=MY,Number=G,Type=Integer,Description="INFO field to test Number=G">
...
... 	T	A	100	PASS	MY=1,3	
```
creates this message: `Line 5: Info MY=1,3 does not match the meta specification Number=G, expected 3 values`.
the inner message is `1,3 does not match the meta specification Number=G, expected 3 values`.
In my opinion, it is already quite clear .